x-pack/plugins/enterprise_search/public/plugin.ts

```diff
@@ -40,29 +40,32 @@ export class EnterpriseSearchPlugin implements Plugin {
     const config = this.config;
 
     core.application.register({
-      id: 'enterprise_search',
-      title: 'App Search', // TODO: This will eventually be 'Enterprise Search' once there's more than just App Search in here
+      id: 'app_search',
+      title: 'App Search',
+      // appRoute: '/app/enterprise_search/app_search', // TODO: Switch to this once https://github.com/elastic/kibana/issues/59190 is in
       category: DEFAULT_APP_CATEGORIES.enterpriseSearch,
       mount: async (params: AppMountParameters) => {
         const [coreStart] = await core.getStartServices();
 
         const { renderApp } = await import('./applications');
+        const { AppSearch } = await import('./applications/app_search');
 
-        return renderApp(coreStart, params, config, plugins);
+        return renderApp(AppSearch, coreStart, params, config, plugins);
       },
     });
+    // TODO: Workplace Search will need to register its own plugin.
 
     plugins.home.featureCatalogue.register({
       id: 'app_search',
       title: 'App Search',
       icon: AppSearchLogo,
       description:
         'Leverage dashboards, analytics, and APIs for advanced application search made simple.',
-      path: '/app/enterprise_search/app_search',
+      path: '/app/app_search', // TODO: Switch to '/app/enterprise_search/app_search' once https://github.com/elastic/kibana/issues/59190 is in
       category: FeatureCatalogueCategory.DATA,
```

Comment:
I'm actually not 100% sure if this will change by the time the "full redesign" is in (CC @johnbarrierwilson) but for right now I think this continues to be the correct category for the home catalog (or at least more correct than admin/management anyway).

![Uploading Screen Shot 2020-05-07 at 3.59.55 PM.png…]()